src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
in Java, array elements are numbered from zero. That is, if we have an array of 10 elements, then the first element of the array will have an index of 0, and the last one will have an index of 9. Therefore, when calling the lengths method, you need to subtract 1 so as not to go beyond the array.
(Colors.values().length-1)